tools/create-cs.sh

```diff
@@ -0,0 +1,20 @@
+ #!/bin/bash
+tag="$1"
+
+[ $# -eq 0 ] && { echo "Usage: $0 <cincinnati-operator-registry image tag>"; exit 1; }
+
+DISCONNECTED_REGISTRY="${DISCONNECTED_REGISTRY:-quay.io/jottofar}"
+
+echo Creating CatalogSource for image ${DISCONNECTED_REGISTRY}/cincinnati-operator-registry:${tag} 
+
+cat <<EOF | oc -n openshift-marketplace create -f -
+kind: CatalogSource
+apiVersion: operators.coreos.com/v1alpha1
+metadata:
+  name: cincinnati-catalog
+spec:
+  sourceType: grpc
+  displayName: Cincinnati Operator
+  publisher: Solutions Engineering
+  image: ${DISCONNECTED_REGISTRY}/cincinnati-operator-registry:${tag}
```

Comment:
Hmm, why separate variables for the repo and tag?  Maybe just:

```sh
  image: ${OPERATOR_IMAGE}
```

and take the whole pullspec from the command line argument?  Or maybe `OPERATOR_REGISTRY_IMAGE` or whatever to reflect whatever this image is supposed to be (some OLM index thingie?).

Reply:
I created these to make it easier to run/test disconnected-cincinnati-operator.md so followed that pattern.